jwala-persistence/src/main/java/com/cerner/jwala/persistence/service/impl/ResourceDaoImpl.java

```diff
@@ -244,6 +248,27 @@ public void updateResource(ResourceIdentifier resourceIdentifier, EntityType ent
         }
     }
 
+
+
+    @Override
+    public void updateResourceGroup(JpaApplication jpaApplication, JpaGroup jpaGroup) {
+        final Query q = em.createNamedQuery(JpaGroupAppConfigTemplate.UPDATE_RESOURCE_GROUP);
+
+        q.setParameter(JpaGroupAppConfigTemplate.QUERY_PARAM_APP, jpaApplication);
+        q.setParameter(JpaGroupAppConfigTemplate.QUERY_PARAM_GRP, jpaGroup);
+        int numEntitities = 0;
+        try {
+            numEntitities = q.executeUpdate();
+
+        } catch (RuntimeException re) {
+            throw new ResourceTemplateUpdateException(jpaApplication, jpaGroup);
+        }
+
+        if (numEntitities == 0) {
+            throw new ResourceTemplateUpdateException(jpaApplication, jpaGroup);
```

Comment:
Add a custom message to this exception stating that the number of updated entities return was zero and that a non-zero number was expected.